content/topics/solo-learn/introduction-to-javascript/4-control-flow/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+content_type: topic
```

Comment:
This topic should be a project because we want to check that the learner has completed this specific course